styles.css

```diff
@@ -0,0 +1,43 @@
+body {
+    background-color: #EE66A6;
+}
+
+h2 {
+    font-size: 1.5rem;
+}
+
+.title {
+    display: flex;
+    justify-content: center;
+}
+
+.card-container {
+    display: grid;
+    grid-template-columns: repeat( auto-fit, minmax(250px, 1fr) );
+    gap: 5px;
+    padding: 20px 20px;
+}
+
+.card {
+    border-radius: 0.25rem;
+    border: 1px solid #00000020;
+    box-shadow: 0 2px 5px 0 rgba(0, 0, 0, 0.2);
+    transition: 200ms ease-in-out;
+    padding: 1rem;
+    background-color: #FFEB55;
+}
+
+.card:hover {
+    transform: scale(1.02);
+}
+
+.card-img {
+    display: block;
+    width: 100%;
+    max-height: 200px;
+    /*aspect-ratio: 16 / 9;*/
+    object-fit: cover;
+    object-position: center;
+    /*border-top-right-radius: 0.25rem;*/
+    /*border-top-left-radius: 0.25rem;*/
```

Comment:
remove commented-out code - if you need it it's safely tucked away in the git history where you can find it again, don't need to keep commented stuff hanging around

Reply:
Done :)